week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -28,76 +28,78 @@ const check = document.getElementById("check");
 //check the right input from forms and if its ok -> add the new book (object in array)
 //via Book function and start render function
 function submit() {
-  if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
-  ) {
-    alert("Please fill all fields!");
-    return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
-    render();
-  }
+	if (
+		title.value == null ||
+		title.value == "" ||
+		author.value == null ||
+		author.value == "" ||
+		pages.value == null ||
+		pages.value == ""
+	) {
+		alert("Please fill all fields!");
+		return false;
+	} else {
+		let book = new Book(title.value, title.value, pages.value, check.checked);
+		library.push(book);
```

Comment:
Have you tested your code? This fails because it refers to variable that doesn't exist, it should be `myLibrary` instead.